src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,65 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { Status } from '../../Enum/Status';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
+type Props = {
+  filter: Status,
+  setFilter: (filter: Status) => void,
+  query: string,
+  setQuery: (query: string) => void,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  filter, setFilter, query, setQuery,
+}) => {
+  const renderSearchIcon = () => (
```

Comment:
It's better to use a semantic HTML tag for the search icon. A button would be appropriate here.